plugins/modules/mso_label.py

```diff
@@ -19,13 +19,12 @@
 - Manage labels on Cisco ACI Multi-Site.
 author:
 - Dag Wieers (@dagwieers)
-version_added: '2.8'
+version_added: '0.0.1'
```

Comment:
If these modules existed in github.com/ansible/ansible then this line can be removed.